snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -82,23 +83,23 @@ init_config:
 #        - TCP
 
 instances:
-  
+
   ## @param ip_address - string - required
   ## For SNMP v1 & v2 enter localhost
-  ## For SNMP v3 enter 192.168.34.10 
+  ## For SNMP v3 enter the IP address of the device to monitor.
```

Comment:
I meant remove the `For SNMP v* ...` and just specify `The IP address of the device to monitor.` for the description